internal/app/osd/main.go

```diff
@@ -11,7 +11,6 @@ import (
 	"github.com/autonomy/talos/internal/app/osd/internal/reg"
```

Comment:
The changes in this file are to remove the old way of generating the PKI. Each application used to generate a certificate. Instead, the `init` now does all PKI bootstrapping and the services use what is handed to them.